task8/ServletsOnMaven/src/main/java/WorkWithDB/WorkWithDB.java

```diff
@@ -0,0 +1,316 @@
+package WorkWithDB;
+
+import Posts.Post;
+import Posts.UtilClasses.EditInfo;
+import Posts.UtilClasses.FilterParams;
+import Posts.UtilClasses.User;
+
+import javax.annotation.Resource;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import javax.xml.crypto.Data;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class WorkWithDB{
+
+
+    public static String getPassword(String username, DataSource dataSource) {
+        try (Connection conn = dataSource.getConnection()) {
+            PreparedStatement statementToGetPassword = conn.prepareStatement(
+                    "select password from users where username = ? ");
+            statementToGetPassword.setString(1, username);
+            ResultSet result = statementToGetPassword.executeQuery();
+            if (result.next()) {
+                return result.getString(1);
+            }
+        }
+        catch(SQLException e) {
+            e.printStackTrace();
```

Comment:
In mature projects, this way of handling exceptions in banned. Use a logger wherever possible